tokio-util/src/cfg.rs

```diff
@@ -18,17 +18,15 @@ macro_rules! cfg_compat {
     }
 }
 
-/*
-macro_rules! cfg_udp {
+macro_rules! cfg_net {
     ($($item:item)*) => {
         $(
-            #[cfg(all(feature = "udp", feature = "codec"))]
-            #[cfg_attr(docsrs, doc(cfg(all(feature = "udp", feature = "codec"))))]
+            #[cfg(all(feature = "net", feature = "codec"))]
+            #[cfg_attr(docsrs, doc(cfg(all(feature = "net", feature = "codec"))))]
```

Comment:
It seems a bit weird that `cfg_net!` is matched under `net + codec`.

Reply:
It was called `cfg_udp` before, I was just sticking with that naming. Does tokio have other `cfg!` macros that have multiple features we can look to for consistent naming?